parcel.go

```diff
@@ -13,48 +13,103 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 }
 
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	// добавление строки в таблицу parcel
+	res, err := s.db.Exec(
+		"INSERT INTO parcel (client, status, address, created_at) " +
+		"VALUES (:client, :status, :address, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("created_at", p.CreatedAt))
+	if err != nil {
+		return 0, err
+	}
+
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+
+	return int(id), nil
 }
 
 func (s ParcelStore) Get(number int) (Parcel, error) {
-	// реализуйте чтение строки по заданному number
+	// чтение строки по заданному number
 	// здесь из таблицы должна вернуться только одна строка
 
-	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
 
+	row := s.db.QueryRow(
+		"SELECT number, client, status, address, created_at FROM parcel WHERE number = :number", 
+		sql.Named("number", number))
+	err := row.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+	if err != nil {
+		return p, err
+	}
+
 	return p, nil
 }
 
 func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
-	// реализуйте чтение строк из таблицы parcel по заданному client
+	// чтение строк из таблицы parcel по заданному client
 	// здесь из таблицы может вернуться несколько строк
 
 	// заполните срез Parcel данными из таблицы
 	var res []Parcel
 
+	rows, err := s.db.Query(
+		"SELECT number, client, status, address, created_at FROM parcel WHERE client = :client",
+		sql.Named("client", client))
+    if err != nil {
+        return res, err
```

Comment:
Если ошибка, то лучше возвращать `nil`, чтобы не были странные данные
```suggestion
        return nil, err
```